cli/opcodes_dictionary.rs

```diff
@@ -0,0 +1,230 @@
+use std::fmt::Display;
```

Comment:
I assume these opcode descriptions were taken from SQLite? This is totally fine as it's in public domain, but please add a comment saying something like:

```
// This source code is derived from SQLite project, which is in public domain:
//
/// https://www.sqlite.org/copyright.html
```

Reply:
done